tools/JsonCli/.azure-pipelines/main.yml

```diff
@@ -31,25 +55,73 @@ jobs:
       command: publish
       publishWebProjects: false
       projects: '$(ProjectPath)'
-      arguments: '--configuration $(BuildConfiguration) --framework netcoreapp3.0'
+      arguments: '--configuration $(BuildConfiguration) --framework netcoreapp3.0 --no-build'
       zipAfterPublish: false
       modifyOutputPath: false
 
+  - task: DeleteFiles@1
+    displayName: 'Delete unneeded publish files'
+    inputs:
+      SourceFolder: '$(SrcPath)'
+      Contents: |
+        bin/**/publish/**/*.exe
+        bin/**/publish/**/*.pdb
+
+  # Run before we build the signing project, because we don't want to analyze that
+  - task: securedevelopmentteam.vss-secure-development-tools.build-task-roslynanalyzers.RoslynAnalyzers@2
+    displayName: 'Run Roslyn Analyzers'
+    continueOnError: true
+    condition: eq(variables['ENABLE_COMPLIANCE'], 'true')
+
+  - task: NuGetCommand@2
+    displayName: 'restore signing project'
+    inputs:
+      restoreSolution: '$(SigningProjectPath)'
+      feedsToUse: config
+      nugetConfigPath: '$(NugetConfigPath)'
+
+  - task: MSBuild@1
+    displayName: 'build signing project'
+    inputs:
+      solution: '$(SigningProjectPath)'
+      configuration: '$(BuildConfiguration)'
+
   - task: CopyFiles@2
-    displayName: 'Copy Files to: $(build.artifactstagingdirectory)'
+    displayName: 'Copy Files to Staging'
     inputs:
       SourceFolder: '$(system.defaultworkingdirectory)/tools/JsonCli/src/'
       Contents: 'bin/**/publish/**'
-      TargetFolder: '$(build.artifactstagingdirectory)'
+      TargetFolder: '$(DropPath)'
     condition: succeededOrFailed()
 
   - task: PublishBuildArtifacts@1
     displayName: 'Publish Artifact: drop'
     inputs:
-      PathtoPublish: '$(build.artifactstagingdirectory)'
+      PathtoPublish: '$(DropPath)'
     condition: succeededOrFailed()
 
-  - template: compliance.yml
+  - task: securedevelopmentteam.vss-secure-development-tools.build-task-binskim.BinSkim@3
+    displayName: 'Run BinSkim'
+    inputs:
+      InputType: Basic
+      AnalyzeTarget: '$(DropPath)\*.dll;$(DropPath)\*.exe'
```

Comment:
Will it matter that the DropPath uses / for path separators and this is using \?

Reply:
The task is smart enough to normalize paths.

https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_build/results?buildId=4091878&view=logs&j=2d2b3007-3c5c-5840-9bb0-2b1ea49925f3&t=9dbcdabf-0886-5e27-5f9c-025a4326930d&l=35